OSDOCS-5243 Adding CLI Manager docs (#81448)
openshift/openshift-docs: cli_reference/cli_manager/cli-manager-using.adoc

:_content-type: ASSEMBLY
[id="cli-manager-install_{context}"]
= Using the {cli-manager}
include::_attributes/common-attributes.adoc[]
:context: cli-manager-using

toc::[]

// Adding a plugin to the CLI Manager

> include::modules/cli-manager-adding-plugin-yaml.adoc[leveloffset=+1]

Reviewer: This is the module I mentioned to pull out into its own assembly for admin tasks

:_mod-docs-content-type: ASSEMBLY

> [id="cli-manager-install_{context}"]

Reviewer 2: Not related to the build failures, but I might suggest changing this module ID so that it's not so close to the `cli-manager-install` ID in the `cli-manager-install.adoc` assembly file:
```suggestion
[id="using-cli-manager_{context}"]
```

Reviewer 2: Oops sorry, I mispoke (this is an assembly also). But still, should drop the `_{context}` from any assembly IDs (and try to match the actual title as close as possible).

= Using the {cli-manager}
include::_attributes/common-attributes.adoc[]
:context: cli-manager-using

toc::[]

> Users can install, update, and uninstall CLI plugins using the {cli-manager} after the cluster administrator has set up and configured the {cli-manager}.

Reviewer 3: ```suggestion
After the cluster administrator sets up and configures the {cli-manager}, users can use it to install, update, and uninstall CLI plugins.
```
